hq/test/logic/RetryTest.scala

```diff
@@ -22,14 +25,14 @@ class RetryTest extends WordSpec with Matchers with AttemptValues {
         body
       }
 
-      Retry.until(testBody, predicateF, failMessage).isFailedAttempt shouldBe true
+      Retry.until(testBody, predicateF, failMessage, testDelay).isFailedAttempt shouldBe true
```

Comment:
If you aren't testing the time delays there's no reason to slow the tests down, why not use `Duration.Zero` for these?

Reply:
👍